charts/dial/examples/azure/complete/values.yaml

```diff
@@ -0,0 +1,197 @@
+core:
+  enabled: true
+  serviceAccount:
+    create: true
+    annotations:
+      azure.workload.identity/client-id: "%%AZURE_WORKLOAD_IDENTITY_CLIENT_ID%%"
+      azure.workload.identity/tenant-id: "%%AZURE_TENANT_ID%%"
```

Comment:
Is it required?

Reply:
Removed tenant-id as it is an optional value